napari/layers/interaction_box.py

```diff
@@ -1,9 +1,44 @@
 import numpy as np
+from skimage.transform import SimilarityTransform
 
 from ._constants import Box
 from ..utils.event import EmitterGroup, Event
 
 
+def inside_boxes(boxes):
```

Comment:
I copied this function from shape_utils.py, because I couldn't import it. As far as I can see this function is not used anywhere else.

Reply:
That's fine, we can remove from Shapes in follow up PR